pkg/terra/median_contract.go

```diff
@@ -20,9 +23,28 @@ const (
 )
 
 // MedianContract interface
+var _ median.MedianContract = (*MedianContract)(nil)
+
+type LatestConfigReader interface {
+	LatestConfigDigestAndEpoch(ctx context.Context) (
+		configDigest types.ConfigDigest,
+		epoch uint32,
+		err error)
+}
+
+type MedianContract struct {
+	contract sdk.AccAddress
```

Comment:
this will read as MedianContract.contract - maybe "address" is more appropriate?